docs/operator-manual/cluster-management.md

```diff
@@ -0,0 +1,23 @@
+# Cluster Management
+
+This guide is for operators looking to manage clusters on the CLI. If you want to use Kubernetes resources for this, check out [Declarative Setup](./declarative-setup.md#clusters).
+
+Not all commands are described here, see the [argocd cluster Command Reference](../user-guide/commands/argocd_cluster.md) for all available commands.
+
+## Adding a cluster
+
+Run `argocd cluster add $CONTEXT`.
```

Comment:
`$CONTEXT` gives the impression that this env has to be set beforehand. I feel using `<context>` or `context-name` is a bit easier to understand.

Reply:
That makes sense, updated it.